Wikipedia/Code/Session.swift

```diff
@@ -59,13 +59,14 @@ public class Session: NSObject {
     }
     
     public struct Callback {
+        public typealias UsedPermanentCache = Bool
         let response: ((URLResponse) -> Void)?
         let data: ((Data) -> Void)?
-        let success: (() -> Void)
+        let success: ((UsedPermanentCache) -> Void)
```

Comment:
I find this `let success: ((UsedPermanentCache) -> Void)` change a little confusing from an in-app API perspective, simply because the `callback.success(false)`/`callback.success(true)` calls themselves convey unclear meaning to me in isolation, but I don't have an alternative to suggest.